python/core/auto_generated/project/qgsproject.sip.in

```diff
@@ -67,6 +67,15 @@ open within the main QGIS application.
       WMSOnlineResource,
     };
 
+    enum ProjectState
```

Comment:
Enums like this should be added to qgis.h -- follow the format of existing enums, including the documentation and capitalisation styles.

Reply:
Agreed